inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -526,6 +529,132 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         return res;
     });
 
+    addSpecificCreator({"LSTMSequence"}, [](const std::shared_ptr<::ngraph::Node>& node,
```

Comment:
Why do you add such creators?

In general case we control the process of conversion. And plugins call all necessary transformations.
I am not sure that you can add tests for such creators without any hacks.

Reply:
I think they are quite useful, if someone has deleted or forgot to add the necessary transformation, then this will help quickly understand the problem.
It is very easy to create a test - just don't call LSTMSequenceToLSTMSequenceIE transformation.

Reply:
In this case:

- First of all, could you add a test?
- The second one, maybe we can provide a special macros in order to generates such creators which throw exception. All these creators have the same code.

Reply:
ok

Reply:
done in a separate PR https://github.com/openvinotoolkit/openvino/pull/1928